### ISSUE TITLE
Regex Expressions Altered in Lexicon Search

### ISSUE THREAD

**Issue by smaugustine** [BetaMasaheft/Documentation#1645]
Regex expressions used in search become altered after clickthroughs, it seems because, while they are initially URL-encoded, they later become decoded and then presumably lost in the process of escaping characters.

Compare:

1. Initial search:

![Screen Shot 2020-11-17 at 19 35 43](https://user-images.githubusercontent.com/6315671/99432190-15fa3e80-290c-11eb-8a84-b2464d67c5b7.png)

2. After clicking through to a result:

![Screen Shot 2020-11-17 at 18 28 33](https://user-images.githubusercontent.com/6315671/99425101-b51a3880-2902-11eb-951e-e4633d120b24.png)

[Copied from [BetaMasaheft/Dillmann #380](https://github.com/BetaMasaheft/Dillmann/issues/380)]

**Comment by PietroLiuzzo** [comment]
may I please ask why this Dillmann question has been copied over here?

**Comment by smaugustine** [comment]
@PietroLiuzzo Are Dillmann issues kept separate? I copied it over because I wasn't sure if Dillmann-related issues, like all the others, were also expected to be grouped here for convenience.

**Comment by PietroLiuzzo** [comment]
no, Dillmann is a separate project, payed by separate funding etc. etc. that is why the issue is there. we do not promote a regex search in BM at the moment.

**Comment by PietroLiuzzo** [comment]
I would like to remove this because it is a copy of an issue already present in its proper place.
